assets/styles/importer.less

```diff
@@ -184,6 +184,14 @@ td h3 {
   color: green;
 }
 
+[ng\:cloak], [ng-cloak], .ng-cloak {
```

Comment:
Did you not figure out a way to make the pokeball keep going till everything was sorted in angular?